developer_setup.md

```diff
@@ -151,6 +151,10 @@
 
   If you do not have Homebrew installed, go to the Homebrew website and install it.
 
+* Install xcode developer tools
+
+```xcode-select --install```
```

Comment:
Does this belong under `Install Packages`?

I know @Fryguy has explained it to me many times, but I can't remember what it does or why it's needed (kernel headers maybe?)

Reply:
I think so, yeah! Headers and build tools and stuff.